edge/pkg/metamanager/process.go

```diff
@@ -128,6 +128,12 @@ func (m *metaManager) processInsert(message model.Message) {
 		return
 	}
 
+	msgSource := message.GetSource()
+	if msgSource == cloudmodules.DeviceControllerModuleName {
+		message.SetRoute(modules.MetaGroup, modules.DeviceTwinModuleName)
+		beehiveContext.Send(modules.DeviceTwinModuleName, message)
+	}
+
 	// Notify edged
 	sendToEdged(&message, false)
```

Comment:
for the device info, we do not need send to edged

Reply:
make it `if else`